pkg/v1/tkg/web/src/app/views/landing/aws-wizard/node-setting-step/node-setting-step.component.ts

```diff
@@ -162,18 +162,18 @@ export class NodeSettingStepComponent extends StepFormDirective implements OnIni
         // key is field name, value is validation rules
         for (const key in this.commonFieldMap) {
             if (key) {
-                this.formGroup.addControl(
+                this.formGroupWithEmition.addControl(
                     key,
                     new FormControl('', this.commonFieldMap[key])
                 );
             }
         }
         this.setControlWithSavedValue(AwsField.NODESETTING_BASTION_HOST_ENABLED, true);
-        this.formGroup.addControl(
+        this.formGroupWithEmition.addControl(
             AwsField.NODESETTING_MACHINE_HEALTH_CHECKS_ENABLED,
             new FormControl(true, [])
         );
-        this.formGroup.addControl(
+        this.formGroupWithEmition.addControl(
```

Comment:
The class is named FormGroupWithEmission, but in all usage you have it spelled with a 't' as Emition. Please change to correct spelling of usages.

Reply:
fixed